src/ray/protobuf/gcs_service.proto

```diff
@@ -452,6 +452,8 @@ message DrainNodeReply {
 
 message InternalKVGetRequest {
   bytes key = 1;
+  // namespace of the key
+  bytes ns = 2;
```

Comment:
```suggestion
  bytes namespace = 2;
```

Here and elsewhere.

Reply:
@ericl I use ns because I thought namespace is a preserved word in cpp and it might not work.
but it looks like protobuf rename this automatically for cpp to `const std::string& namespace_() const;`
I'll update this.